x-pack/plugins/security_solution/server/lib/detection_engine/rules/saved_object_mappings.ts

```diff
@@ -53,3 +53,26 @@ export const type: SavedObjectsType = {
   namespaceType: 'single',
   mappings: ruleStatusSavedObjectMappings,
 };
+
+export const ruleAssetSavedObjectType = 'security-rule';
```

Comment:
@FrankHassanabad is this change okay?
for some reason, it doesn't look like fleet has recognized it yet when running local tests

Reply:
Should be ok. Don't see anything weird here.